src/Tizen.NUI/src/public/Renderer.cs

```diff
@@ -47,12 +47,16 @@ public int DepthIndex
             get
             {
                 int temp = 0;
-                Tizen.NUI.Object.GetProperty(SwigCPtr, Renderer.Property.DepthIndex).Get(out temp);
+                Tizen.NUI.PropertyValue pValue = Tizen.NUI.Object.GetProperty(SwigCPtr, Renderer.Property.DepthIndex);
+                pValue?.Get(out temp);
```

Comment:
To keep the same logic as before, please remove `?`.
Tizen.NUI.Object.GetProperty is returning `new PropertyValue` which can not be null.

Reply:
> To keep same logic as before, please remove `?`.
> Tizen.NUI.Object.GetProperty is returning `new PropertyValue` which can not be null.

Fixed.